google/cloud/documentai_toolbox/converters/vision_helpers.py

```diff
@@ -226,6 +227,52 @@ def _convert_document_token(
         page_info.token_idx += 1
     return vision_words
 
+def _generate_entity_annotations(
+    page_info: PageInfo,
+) -> List[EntityAnnotation]:
+  """Generate a list of EntityAnnotations from Document.
+
+  Args:
+    page_info: Current page information, including document page to be converted
+      , its text, and the position of reading cursor.
+
+  Returns:
+    A list of EntityAnnotations with descriptions and bounding box populated. A
+    EntityAnnotation has a word level information.
+  """
+  text_annotations = []
+  for token in page_info.page.tokens:
+    v = []
```

Comment:
Could you change this `v` list to use the `Vertex` type for Vision? Just to maintain type safety